### PR TITLE
#1559 Move files from / to utils folder or where it fits best

### LINKED ISSUE #1559: Move most of stuff from / to some utils folder or where it fits best
Let's move most of the stuff files which don't have strong requirements to be in / somewhere else so users visit our "frontpage" readme.md doesn't need to scroll through like Eiffel tower 

TIL https://en.wikipedia.org/wiki/Gustave_Eiffel
https://en.wikipedia.org/wiki/Eiffel_Tower

current status
image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new?
- [x] PR closes #1559
- [ ] Moved most of the files in utils folder, also had a `types.ts` file, was not sure about where to move this so for now it is also in utils. 

### Had issue bounty label ?
- [x] Fill up your KSM address: 
[Payout](https://kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [ ] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
- What do you think about moving `i18n.ts` to `config/i18n.ts` ?
- relative paths kinda scare me 😨

**Comment by prachi00** [comment]
> * moving

so you mean create new folder called config and move i18n there?

**Comment by roiLeo** [comment]
> > * moving
> 
> so you mean create new folder called config and move i18n there?

yes that's right

**Comment by prachi00** [comment]
> > > * moving
> > 
> > 
> > so you mean create new folder called config and move i18n there?
> 
> yes that's right

@roiLeo but wouldn't that introduce more folders in '/' , hence more scroll is required

**Comment by roiLeo** [comment]
> @roiLeo but wouldn't that introduce more folders in '/' , hence more scroll is required

We can try `utils/config/..`, what do yout think?

**Comment by prachi00** [comment]
> > @roiLeo but wouldn't that introduce more folders in '/' , hence more scroll is required
> 
> We can try `utils/config/..`, what do yout think?

yes this sounds good, I'll do this

File: utils/pinata.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Axios from 'axios'
-import { extractCid, justHash } from './utils/ipfs'
+import { extractCid, justHash } from './ipfs'
```
Comment (roiLeo) [inline utils/pinata.ts:2]: - relative path

File: utils/proxy.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import Axios from 'axios'
-import { NFT, NFTMetadata } from './components/rmrk/service/scheme'
+import { NFT, NFTMetadata } from '../components/rmrk/service/scheme'
 import { APIKeys, pinFile as pinFileToIPFS } from './pinata'
-import { extractCid, justHash } from './utils/ipfs'
+import { extractCid, justHash } from './ipfs'
```
Comment (roiLeo) [inline utils/proxy.ts:4]: - relative path

**Review by roiLeo — APPROVED** [review]

**Review by vikiival — APPROVED** [review]
Legit

**Comment by yangwao** [comment]
@prachi00 some merge conflicts has appeared after merging #1556 
Let's fix them so we can close on this one :)

**Comment by prachi00** [comment]
> @prachi00 some merge conflicts has appeared after merging #1556 Let's fix them so we can close on this one :)

@yangwao have fixed the conflicts

**Comment by yangwao** [comment]
Merry! 

Perfect, I’ve sent the payout 
https://kusama.subscan.io/extrinsic/0x2aa5eff6b573fa85eb3ddce5c63736c3ab167745781b4291a5b897eebebadabe
Let’s grab another issue and get rewarded! 
https://github.com/kodadot/nft-gallery/issues/

**Pull request merged**